source/common/stats/stats_impl.h

```diff
@@ -500,11 +560,11 @@ class IsolatedStoreImpl : public Store {
     const std::string prefix_;
   };
 
-  HeapRawStatDataAllocator alloc_;
+  StatsOptionsImpl stats_options_;
```

Comment:
nit: `const`?

Reply:
no longer const-able; I need to update it after allocating it, in order to set the truncation limit high.